GameServerLib/Config.cs

```diff
@@ -90,6 +97,30 @@ private void LoadConfig(Game game, string json)
             MapData = ContentManager.GetMapData(GameConfig.Map);
             MapSpawns = ContentManager.GetMapSpawns(GameConfig.Map);
         }
+
+        private string getContentPath()
+        {
+            string result = null;
+            var path = new DirectoryInfo(Directory.GetCurrentDirectory());
+
+            while (result == null) {
+                if (path == null) {
```

Comment:
We mostly use https://en.wikipedia.org/wiki/Indentation_style#Allman_style for brackets in the code. In this function you have two different code styles for bracket indentation. I think that it would be better to just stick with https://en.wikipedia.org/wiki/Indentation_style#Allman_style

Reply:
I Will do that, would be cool if someone could commit code-style format settings.

Reply:
This is actually issue #908. I think only the admin can add bots to the repo. There is also Github actions, but it never worked for me.

Reply:
Oh, what I meant was just the settings file for rider or vs, but that would've sounded good too.